hypervisor/lazy.go

```diff
@@ -49,6 +49,13 @@ func statePreparing(ctx *VmContext, ev VmEvent) {
 		ctx.setWindowSize(cmd.ClientTag, cmd.Size)
 	case COMMAND_RUN_POD, COMMAND_REPLACE_POD:
 		glog.Info("got spec, prepare devices")
+		err := ctx.DCtx.(LazyDriverContext).InitVM(ctx)
```

Comment:
I picked the `initVM` from `LazyVmLoop` as a public interface, this function is used to create a register a VBox VM.  Since we have to make sure the vm is created before adding a new port forward and adding a new network interface in `lazyPrepareDevice`.

> And for the design, run pod only create pod,not create vm.

This fix should not break the design, the `initVM` is located in `LazyLaunch` function, I did only re-order them.

What do you think? @gao-feng